doc/level1.rst

```diff
@@ -637,29 +639,45 @@ Key Gossip
 ++++++++++
 
 It is a common use case to send an encrypted mail to a group of
-recipients. To ensure that these recipients can encrypt messages when
-replying to that same group, the keys of all recipients can be
-included in the encrypted payload. This does not include BCC
+recipients. To help these recipients to send encrypted replies
```

Comment:
what's wrong with "ensure"? it's really what is happening, isn't it?

Reply:
there are some minor exceptions (e.g. replying to messages after a key has expired, even though the original message was sent before expiry), but they're narrow enough that i think "ensure" is acceptable here.